src/many/src/server/module/_9_account/features/multisig.rs

```diff
@@ -0,0 +1,233 @@
+        }
+
+        match argument.get(0) {
+            Some(CborAny::Map(m)) => {
+                let threshold = m.get(&CborAny::Int(0)).and_then(|v| match v {
+                    CborAny::Int(x) => (*x).try_into().ok(),
+                    _ => None,
+                });
+                let timeout_in_secs = m.get(&CborAny::Int(1)).and_then(|v| match v {
+                    CborAny::Int(x) => (*x).try_into().ok(),
+                    _ => None,
+                });
+                let execute_automatically = m.get(&CborAny::Int(2)).and_then(|v| match v {
+                    CborAny::Bool(x) => Some(*x),
+                    _ => None,
+                });
+
+                Ok(Self {
+                    arg: MultisigAccountFeatureArg {
+                        threshold,
+                        timeout_in_secs,
+                        execute_automatically,
+                    },
+                })
+            }
+            _ => Err(ManyError::invalid_attribute_arguments()),
+        }
+    }
+}
+
+impl super::FeatureInfo for MultisigAccountFeature {
+    fn as_feature(&self) -> Feature {
+        let mut map = BTreeMap::<CborAny, CborAny>::new();
+        if let Some(threshold) = self.arg.threshold {
+            map.insert(CborAny::Int(0), CborAny::Int(threshold as i64));
+        }
+        if let Some(timeout_in_secs) = self.arg.timeout_in_secs {
+            map.insert(CborAny::Int(1), CborAny::Int(timeout_in_secs as i64));
+        }
+        if let Some(execute_automatically) = self.arg.execute_automatically {
+            map.insert(CborAny::Int(2), CborAny::Bool(execute_automatically));
+        }
+
+        Feature::with_id(Self::ID).with_argument(CborAny::Map(map))
+    }
+
+    fn roles() -> BTreeSet<String> {
+        BTreeSet::from([
+            "canMultisigSubmit".to_string(),
+            "canMultisigApprove".to_string(),
+        ])
+    }
+}
+
+#[derive(Clone, Encode, Decode)]
+#[cbor(map)]
+pub struct SubmitTransactionArg {
+    #[n(0)]
+    pub account: Option<Identity>,
+
+    #[n(1)]
+    pub memo: Option<String>,
+
+    #[n(2)]
+    pub transaction: TransactionInfo,
+
+    #[n(3)]
+    pub threshold: Option<u64>,
+
+    #[n(4)]
+    pub timeout_in_secs: Option<u64>,
+
+    #[n(5)]
+    pub execute_automatically: Option<bool>,
+}
+
+#[derive(Clone, Encode, Decode)]
+#[cbor(map)]
+pub struct SubmitTransactionReturn {
+    #[n(0)]
+    pub token: ByteVec,
+}
+
+#[derive(Clone, Encode, Decode)]
+#[cbor(map)]
+pub struct InfoArg {
+    #[n(0)]
+    pub token: ByteVec,
+}
+
+#[derive(Clone, Default, Encode, Decode)]
+#[cbor(map)]
+pub struct ApproverInfo {
+    #[n(0)]
+    pub approved: bool,
+}
+
+#[derive(Clone, Encode, Decode)]
+#[cbor(map)]
+pub struct InfoReturn {
+    #[n(0)]
+    pub memo: Option<String>,
+
+    #[n(1)]
+    pub submitter: Identity,
+
+    #[n(2)]
+    pub transaction: TransactionInfo,
+
+    #[n(3)]
+    pub approvers: BTreeMap<Identity, ApproverInfo>,
```

Comment:
You should define `ApproverInfo` in the spec. It's a `bool` in the spec. Here it's a map.

Reply:
It's not a bool, it's

```
{
            ; Whether they approved or not.
            0 => bool
        }
```

Doesn't have a type name in the spec.